routers/api/v1/notify/notifications.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package notify
+
+import (
+	"net/http"
+
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/context"
+	api "code.gitea.io/gitea/modules/structs"
+)
+
+// NewAvailable check if unread notifications exist
+func NewAvailable(ctx *context.APIContext) {
+	// swagger:operation GET /notifications/new notification notifyNewAvailable
+	// ---
+	// summary: Check if unread notifications exist
+	// responses:
+	//   "204":
+	//     description: No unread notification
+	//   "302":
+	//    "$ref": "#/responses/NotificationCount"
+
+	count := models.CountUnread(ctx.User)
+
+	if count > 0 {
+		ctx.JSON(http.StatusFound, api.NotificationCount{New: count})
```

Comment:
Rethinking of it this should definitly not be StatusFound
```suggestion
		ctx.JSON(http.StatusOK, api.NotificationCount{New: count})
```

Reply:
Or maybe keep StatusFound with the Location header set to the full content of notification.

Reply:
hmm searchin ...

Reply:
ok in the 200 range OK is probably the best